scorm-tool/src/java/org/sakaiproject/scorm/ui/console/pages/ConsoleBasePage.java

```diff
@@ -84,8 +83,15 @@ public boolean isVisible()
 		// add the toolbar container
 		add(navBar);
 
+		// page title
 		add(newPageTitleLabel(params));
-		add(feedback = new SakaiFeedbackPanel("feedback"));
+
+		// feedback panel
+		feedback = new SakaiFeedbackPanel("feedback");
```

Comment:
You can chain this if you want: add(feedback = new SakaiFeedbackPanel("feedback").setOutputMarkupId(true));

Reply:
In this case, I can't without adding some casting, because `setOutputMarkupId(true)` returns a `Component`, but the assignment to `feedback =` is expecting a `SakaiFeedbackPanel` concrete `Component`